02week/pigLatin.js

```diff
@@ -7,11 +7,35 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
-
+//my code
 function pigLatin(word) {
+  //This code stores the list of vowels so that the words can be searched for vowels.
+  const vowelArray = ['a', 'e', 'i', 'o', 'u'];
+
+  //This code is to trim the word and change it all to lowercase for the sake of testing.
+  word = word.toLowerCase().trim();
 
-  // Your code here
+  //This code splits the string into individual letters so that each letter can be looped individually.
+  let wordArray = word.split('');
+
+  let counter = 0;
+  while(!isAVowel(wordArray[0])) {
+    wordArray.push(wordArray.shift());
+    counter++;
+  }
+  if(counter === 0) {
+    return wordArray.join('') + 'yay';
+  } else {
+    return wordArray.join('') + 'ay';
+  }
+}
 
+const isAVowel = (letter) => {
+  if(letter === 'a' || letter === 'e' || letter === 'i' || letter === 'o' || letter === 'u') {
```

Comment:
instead of a strict check, a cleaner javascript method would be to check if the letter is in the vowelArray. I would use "indexOf" to see if the letter exists in the vowel array.